src/data/markdown/translated-guides/en/05 Test Types/02 Load Testing.md

```diff
@@ -4,157 +4,109 @@ head_title: 'What is Load Testing? How to create a Load Test in k6'
 excerpt: 'A Load Test is a type of Performance Test that is primarily concerned with assessing the performance of your system in terms of concurrent users or requests per second. Let’s see an example.'
 ---
 
-Load Testing is primarily concerned with assessing the current performance of your system in terms
-of concurrent users or requests per second.
 
-When you want to understand if your system is meeting the performance goals, this is the type of test you'll run.
+An average-load test assesses how the system performs under typical load. Typical load might be a regular day in production or an average moment.
 
-> #### What is Load Testing
->
-> Load Testing is a type of Performance Testing used to
-> determine a system's behavior under both normal and peak conditions.
->
-> Load Testing is used to ensure that the application performs satisfactorily
-> when many users access it at the same time.
+Average-load tests try to simulate the number of concurrent users and requests per second that reflect average behaviors in the production environment.This type of test typically increases the throughput or VUs gradually and keeps that average amount of load for some time. Depending on the system's characteristics, the test may stop suddenly or have a short ramp-down period.
 
-You should run a Load Test to:
+![Overview of an average load test](images/chart-average-load-test-overview.png)
 
-1.  Assess the current performance of your system under typical and peak load.
-2.  Make sure you continue to meet the performance standards as you make changes to your system (code and infrastructure).
+Since “load test” might refer to all types of tests that simulate traffic, this guide uses the name _average-load test_ to avoid confusing.
```

Comment:
```suggestion
Since “load test” might refer to all types of tests that simulate traffic, this guide uses the name _average-load test_ to avoid confusion.
```